src/ads.test.ts

```diff
@@ -1,11 +1,13 @@
-import { insertAdPlaceholders } from './ads';
+import { insertAdPlaceholdersWhenEnabled } from './ads';
 import { ReactNode } from 'react';
 import { renderAll } from 'renderer';
 import { JSDOM } from 'jsdom';
 import { Pillar } from 'pillar';
 import { compose } from 'lib';
 import { ElementKind, BodyElement } from 'item';
 
+const insertAdPlaceholders = insertAdPlaceholdersWhenEnabled(false);
```

Comment:
This might just be me, but I'm reading this as "ad placeholders are not enabled because `false` has been passed". However, as far as I understand it, passing `false` will guarantee that they _are_ enabled.

Would it make sense to flip this boolean inside the function, or tweak this function name?

Reply:
yeah it caught me out too https://github.com/guardian/apps-rendering/pull/201/commits/902fd782682faaf25c76751771816fb689617ddc

I will change